src/modules/45-dashboards/components/DashboardCard/DashboardCard.tsx

```diff
@@ -42,6 +40,15 @@ const DashboardCard: React.FC<DashboardCardProps> = ({
   const { getString } = useStrings()
   const { accountId, folderId } = useParams<{ accountId: string; folderId: string }>()
   const [menuOpen, setMenuOpen] = useState(false)
+  const { editableFolders } = useDashboardsContext()
+
+  const isCloneable: boolean = React.useMemo(() => {
+    return editableFolders ? editableFolders?.length > 0 : false
```

Comment:
You can simplify this a wee bit
```suggestion
    return !!editableFolders?.length
```